tools/build.py

```diff
@@ -6,15 +6,14 @@
 from fontTools import subset
 from io import StringIO
 from pcpp.preprocessor import Preprocessor
-from sfdLib.parser import SFDParser
-from sfdLib.utils import GLYPHCLASS_KEY, MATH_KEY
+from sfdLib.parser import SFDParser, CATEGORIES_KEY, MATH_KEY
```

Comment:
Is this commit a copy of my work getting this building again or did you do this separately and I missed it?

Reply:
Yes, this is https://github.com/alerque/libertinus/commit/dc071e5f18d3e729698e1d49aa6f0bdfe30af660. I pulled this into this branch so I could work with an up to date fontship installation

Reply:
Okay. With that in mind next time the way to do that would be with `git merge` or `git cherry-pick`. No worries I can correct it. I'll post an update to this PR in a minute that should pull this in the right way and help Git understand there isn't a conflict.

Reply:
I've corrected the merge issues (this branch and the master branch both add characters, so I bumped the ones in this PR up to the new available slots). You don't seem to have enabled the checkbox for allowing maintainers to push to your branch though, so I can't update it here. You can either check of that box in the PR and I'll fix it here, or you can fix it on your end like this (I pushed to a feature branch on the repo that you can switch to via cebf997):

```console
$ git checkout E-with-stroke
$ git fetch --all
$ git reset --hard cebf997
$ git push --force
```

Reply:
Thanks, Caleb! I've been looking for the "allow edits from maintainers" button, but was unable to find it. I will look into this further, maybe it has something to do with the fact that I'm pulling from a repo owned by an organisation, not a single user...

Reply:
@kopeckyf the checkbox should appear below the main textarea when creating the PR, and on the sidebar afterwards. For example, here's what I see in one of the PRs I authored:

![Screenshot 2021-05-25 at 16 58 18](https://user-images.githubusercontent.com/478237/119530205-b5483480-bd7a-11eb-8586-0f704b8604d3.png)

Reply:
Thanks, @waldyrious, but that button is missing in this PR :unamused:

Reply:
Ah, maybe it's because the PR comes from a fork in the @langsci organization, rather than under your own account. Apparently lots of people have bumped into this, too: https://github.com/isaacs/github/issues/1681.

Reply:
Thanks for the link! I will speak to my organisation about adding @alerque as a collaborator in our Libertinus fork.